server/py/services/api/tests/unit/crud/test_runs.py

```diff
@@ -935,6 +936,20 @@ def _validate_run_artifacts(
         iter=0,
         run_format: mlrun.common.formatters.RunFormat = None,
     ):
+        def _normalize_datetime_fields(artifact):
```

Comment:
```suggestion
        def normalize_datetime_fields(artifact):
```

no need to underscore a nested function. also better of adding a one-liner comment atop function of what is the function actually doing. e.g "Ensures timestamp has milliseconds" or so